src/callbacks_step/averaging_dg.jl

```diff
@@ -0,0 +1,51 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+function save_averaging_file(averaging_callback, mesh::TreeMesh, equations, dg::DGSEM, cache)
+  @unpack output_directory, filename, mean_values = averaging_callback
+  h5open(joinpath(output_directory, filename), "w") do file
+    # Add context information
+    attributes(file)["ndims"] = ndims(mesh)
+    attributes(file)["polydeg"] = polydeg(dg)
+    attributes(file)["nelements"] = nelements(dg, cache)
```

Comment:
I like the idea. Right now, it is possible to visualize the mean values by running an acoustics simulations and saving the solution using `cons2mean`. It would be a lot nicer to use the averaging files directly, but I think there is a bit of effort required to make it work and in addition, the whole CFD/CAA code is still in an experimental stage so I don't think it's worth it the effort right now. 
My suggestion is to track this in an issue once this PR is merged. What do you think?

Reply:
Sounds good to me! If you keep track of this somewhere else, feel free to resolve this conversation.